sktime/forecasting/compose/_pipeline.py

```diff
@@ -236,11 +236,39 @@ class ForecastingPipeline(_Pipeline):
     to X. The forecaster can also be a TransformedTargetForecaster containing
     transformers to transform y.
 
+    For a list `t1`, `t2`, ..., `tN`, `f`, `tp1`, `tp2`, ..., `tpM`
+    where `t[i]` and `tp[i]` are transformers (`t` to pre-, `tp` to post-process), and
+    `f` is an sktime forecaster, the pipeline behaves as follows:
+
+    `fit(y, X, fh)`  - If `X` is not given then it just passthrough the data without
+    transformation. Otherwise, it clones `t1` by `t1.clone()`, changes state by running
+    t1.fit_transform` with `X=X` and `y=y`, then clones `t2` and run `t2.fit_transform`
+    with `X`= the output of `tp1.fit_transform` etc, sequentially, with
+    `tp[i]` receiving the output of `tp[i-1]`,
+
+    `predict(fh, X)` - The result is of running `t1.transform` with `X=X` then running
+    `f.predict` with `fh=fh` and `X` = the output of `tp1.transform`, etc., sequentially
+    with `tp[i]` receiving the output of `tp[i-1]`,
+
+    `predict_interval(X, fh)`, `predict_quantiles(X, fh)` - as `predict(X, fh)`, with
+    `predict_interval` or `predict_quantiles` substituted for `predict`, `predict_var`,
+    `predict_proba` - uses base class default to obtain crude estimates
+    from `predict_quantiles`.
+    Recommended to replace with better custom implementations if needed.
```

Comment:
I think this line should be removed, since it is not clear *where* the "better custom implementation" should be located.